### PR TITLE
TODO: code cleanup.

### PR DESCRIPTION
**Pull request by Nickelony**
I'm just cleaning up.

Ready to review:
+ shavit-core

### CODE REVIEW

File: addons/sourcemod/scripting/shavit-core.sp [hunk 1]
```diff
@@ -59,40 +59,40 @@ Handle gH_Forwards_OnDatabaseLoaded = null;
...
-float gF_AngleCache[MAXPLAYERS+1];
-int gI_TotalMeasures[MAXPLAYERS+1];
-int gI_GoodGains[MAXPLAYERS+1];
-bool gB_DoubleSteps[MAXPLAYERS+1];
-float gF_StrafeWarning[MAXPLAYERS+1];
-bool gB_PracticeMode[MAXPLAYERS+1];
-int gI_SHSW_FirstCombination[MAXPLAYERS+1];
-int gI_Track[MAXPLAYERS+1];
+// Timer variables
+bool gB_TimerEnabled[MAXPLAYERS + 1];
```
Comment (shavitush) [inline addons/sourcemod/scripting/shavit-core.sp]: I don't like the change from `x+1` to `x + 1`

File: addons/sourcemod/scripting/shavit-core.sp [hunk 2]
```diff
@@ -114,28 +114,28 @@ bool gB_BlockPreJump = false;
...
+// Chat settings
 char gS_ChatStrings[CHATSETTINGS_SIZE][128];
 
-// misc cache
+// Misc cache
 bool gB_StopChatSound = false;
 bool gB_HookedJump = false;
 
-// kz support
+// Kz support
```
Comment (shavitush) [inline addons/sourcemod/scripting/shavit-core.sp]: If anything, "KZ" rather than "Kz"

File: addons/sourcemod/scripting/shavit-core.sp [hunk 3]
```diff
@@ -226,26 +249,18 @@ public void OnPluginStart()
 	{
 		SetFailState("This plugin was meant to be used in CS:S, CS:GO and TF2 *only*.");
```
Comment (shavitush) [inline addons/sourcemod/scripting/shavit-core.sp]: This is completely wrong and breaks the whole plugin for CS:S and TF2. Also, this function should also handle `sv_autobunnyhopping` as it's CS:GO specific.

File: addons/sourcemod/scripting/shavit-core.sp [hunk 4]
```diff
@@ -1648,10 +1667,10 @@ void SQL_SetPrefix()
 	{
 		SetFailState("Cannot open \"configs/shavit-prefix.txt\". Make sure this file exists and that the server has read permissions to it.");
 	}
-	
-	char[] sLine = new char[PLATFORM_MAX_PATH*2];
 
-	while(fFile.ReadLine(sLine, PLATFORM_MAX_PATH*2))
+	char[] sLine = new char[PLATFORM_MAX_PATH * 2];
```
Comment (shavitush) [inline addons/sourcemod/scripting/shavit-core.sp]: Again, I don't think the change from `x*2` to `x * 2` is necessary.

File: addons/sourcemod/scripting/shavit-core.sp [hunk 5]
```diff
@@ -1648,10 +1667,10 @@ void SQL_SetPrefix()
...
+
+	while(fFile.ReadLine(sLine, PLATFORM_MAX_PATH * 2))
```
Comment (shavitush) [inline addons/sourcemod/scripting/shavit-core.sp]: ^

File: addons/sourcemod/scripting/shavit-core.sp [hunk 6]
```diff
@@ -2159,11 +2064,169 @@ public Action OnPlayerRunCmd(int client, int &buttons, int &impulse, float vel[3
...
+
+	if(++cycle % 50 == 0)
+	{
+		Shavit_StopChatSound();
+		Shavit_PrintToChat(client, "vel[0]: %.01f | vel[1]: %.01f", vel[0], vel[1]);
+	}
+}
+#endif
+
+void BlockButtons(int client, int buttons, float vel[3], int iGroundEntity)
```
Comment (shavitush) [inline addons/sourcemod/scripting/shavit-core.sp]: `int buttons` >> `int &buttons`

`iGroundEntity` >> `groundentity`, as it's a parameter. should be consistent with the rest of the code

File: addons/sourcemod/scripting/shavit-core.sp [hunk 7]
```diff
@@ -2159,11 +2064,169 @@ public Action OnPlayerRunCmd(int client, int &buttons, int &impulse, float vel[3
...
+
+		if(gA_StyleSettings[gBS_Style[client]][bBlockD] && ((buttons & IN_MOVERIGHT) > 0 || vel[1] > 0.0))
+		{
+			vel[1] = 0.0;
+			buttons &= ~IN_MOVERIGHT;
+		}
+	}
+}
+
+void HandleHSWStyles(int client, int buttons, float vel[3], int iGroundEntity, bool bInStart)
```
Comment (shavitush) [inline addons/sourcemod/scripting/shavit-core.sp]: `int buttons` >> `int &buttons`
`iGroundEntity` >> `groundentity`
`bInStart` >> `instart`

**Comment by shavitush** [comment]
I'll try and push a few changes to `shavit-core` this week, so this pull request is very likely to break. Just a heads up! Solving the conflict(s) if there will be any, shouldn't be hard at all.

**Comment by Nickelony** [comment]
GitHub hates me... I'll just open another PR.

**Pull request closed without merging**